.eslintrc.js

```diff
@@ -0,0 +1,74 @@
+module.exports = {
```

Comment:
@micheal-parks I copied this as-is from three-inspect - let me know if any there are any settings that don't apply here.